src/Discord.Net.Rest/API/Common/AuditLogOptions.cs

```diff
@@ -4,11 +4,12 @@ namespace Discord.API
 {
     internal class AuditLogOptions
     {
-        //Message delete
         [JsonProperty("count")]
-        public int? MessageDeleteCount { get; set; }
+        public int? Count { get; set; }
         [JsonProperty("channel_id")]
-        public ulong? MessageDeleteChannelId { get; set; }
```

Comment:
Why are these properties renamed and that comment removed?

Reply:
Looking at the code, probably because they're re-used for multiple audit log events. This isn't actually a breaking change as it's an internal type.

Reply:
For context, when I initially made this type, only message deletes had `channel_id` and `count` in the `options` object in each audit log entry.